### ISSUE TITLE
Bundle.zip download: connection problem

### ISSUE THREAD

**Issue by ArtemSopin** [pophealth/popHealth#275]
Hello,
I cannot download the bundle zip , it shows me connection problem. Is this server still available? 

C:\Program Files (x86)\cURL\bin>curl -u <username>:<password> http://demo.projectcypress.org/bundles/bundle-latest.zip -o ../bundle-latest.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:12 --:--:--     0curl: (7) Failed to connect to demo.projectcypress.org port 80: Timed out

**Comment by quzhouxiachuan** [comment]
I have the same issue, mine shows connection time out

failed to connect to demo.projectcypress.org port 80

**Comment by benhmin** [comment]
I don't think that server exists anymore. They have changed to https://cypressdemo.healthit.gov/measure_bundles/ and no longer have a bundle_latest.zip
